MigrateDefaultAspNetIdentityToAdminUiSchema.sql

```diff
@@ -81,7 +69,7 @@ BEGIN TRANSACTION TransactionOne
 		) ON [PRIMARY]
 	END
 
-	INSERT INTO [dbo].[__EFMigrationsHistory]
+	INSERT INTO [__EFMigrationsHistory]
```

Comment:
If this is bringing the AspNetIDenttiy migration script to the current schema version, we will need to make sure we add all of the migration histories so the migrations don't fail. It's probably worth testing this with the Identity migration to see if everything still works.